packages/web/src/components/spa/HackAUBG/RegistrationForm/RegistrationForm.jsx

```diff
@@ -0,0 +1,731 @@
+import React, { useEffect } from 'react'; // eslint-disable-line
+import './registration_form.css';
+import { useForm } from 'react-hook-form';
+import { useState } from 'react';
+import axios from 'axios';
+import { url } from '../../../../Global';
+
+const RegistrationForm = () => {
+    const {
+        register,
+        handleSubmit,
+        setError,
+        formState: { errors } // eslint-disable-line
+    } = useForm({ mode: 'all' });
+
+    const [loadingAnimation, setLoadingAnimation] = useState(false);
+    const [submitPressed, setSubmitPressed] = useState(false);
+    const [submitButtonValue, setSubmitButtonValue] = useState('register'); // eslint-disable-line
+    const [apiError, setApiError] = useState(false);
+
+    const makeRegistration = (data) => {
+        axios({
+            method: 'post',
+            url: url + '/api/hackathon/register',
+            headers: {
+                'BEARER-TOKEN': localStorage.getItem('auth_token')
+            },
+            data: { ...data }
```

Comment:
you don't need the three dots here, no need to unpack, just pass data without the brackets